install.ps1

```diff
@@ -23,7 +23,10 @@ Install-ChocoPackage python2
 Install-ChocoPackage nodejs-lts
 
 # Installing Git
-choco install -y git
+Install-ChocoPackage git
+$env:ChocolateyInstall = Convert-Path "$((Get-Command choco).path)\..\.."
```

Comment:
Move lines 27 and 28 up to line 6. This should be done right after choc installation so everything else in the script can use Update-SessionEnvironment if they need to.

Reply:
ok